upstream svn31350 fix dvips
---
 texk/dvipsk/ChangeLog |    5 +++++
 texk/dvipsk/bbox.c    |    4 ++--
 2 files changed, 7 insertions(+), 2 deletions(-)

 2013-04-07  Karl Berry  <karl@tug.org>
 
 	* resident.c (getdefaults) <m>: if <= 0, set to INT_MAX.
--- texk/dvipsk/bbox.c
+++ texk/dvipsk/bbox.c
@@ -346,8 +346,8 @@
             if (v - bcd->ury < lly) lly = v - bcd->ury;
             if (v - bcd->lly > ury) ury = v - bcd->lly;
          } else {
-            if (h + bcd->ury < llx) llx = h + bcd->ury;
-            if (h + bcd->lly > urx) urx = h + bcd->lly;
+            if (h + bcd->ury > urx) urx = h + bcd->ury;
+            if (h + bcd->lly < llx) llx = h + bcd->lly;
             if (v + bcd->llx < lly) lly = v + bcd->llx;
             if (v + bcd->urx > ury) ury = v + bcd->urx;
          }
